src/main/jbake/content/blog/2017/03/24/distributed-tracing-with-camel.adoc

```diff
@@ -0,0 +1,267 @@
+= Distributed Tracing with Apache Camel and OpenTracing
+Gary Brown
+2017-03-24
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, opentracing, apache, camel
+
+In this post we will discuss how the http://opentracing.io/[OpenTracing standard] is being supported in the upcoming release of Apache Camel (version 2.19).
```

Comment:
Perhaps mention that you want to show than on 2 impls : existing Zipkin + new OT (with H-APM) ?